### PR TITLE
Externalize more hardcoded values

### PR DESCRIPTION
**Pull request by lynxlynxlynx**
Lifted, adapted and extended the (IMHO) last useful things from #1002.

### CODE REVIEW

File: rust/stracciatella/src/schemas/yaml/game.schema.yaml [hunk 1]
```diff
@@ -234,6 +241,41 @@ properties:
       Multiplier for damage that is inflicted on the legs
 
       Vanilla setting: `0.5`
+  aim_bonus_per_std_ap:
+    type: number
```
Comment (ja2-derek) [inline rust/stracciatella/src/schemas/yaml/game.schema.yaml:245]: I think we should use `$ref: types/int8.schema.yaml` for these, to guard against overflow - if somebody decides they want a really high value for these.

Comment (lynxlynxlynx) [reply rust/stracciatella/src/schemas/yaml/game.schema.yaml:245]: That's a design question to me: where do we draw the line? If they want to break the file, the game, they have countless options already.

Comment (ja2-derek) [reply rust/stracciatella/src/schemas/yaml/game.schema.yaml:245]: Maybe always use the same numeric value as the backing C++ code?

I was looking at `range_bonus_barrel_extender`. The default is `100`. It is not unreasonable to raise it to say `150`, and that already causes overflow.

Comment (lynxlynxlynx) [reply rust/stracciatella/src/schemas/yaml/game.schema.yaml:245]: Good point, it should be bigger on the engine side already.

**Comment by ja2-derek** [comment]
It looks good to me now.

Now that I think more about it, the next thing I want (in the future in another proposal) is to have it generalized. The aim penalties and bonuses should have been properties of the item, so a mod have more varieties of weapon attachments.

**Pull request merged**